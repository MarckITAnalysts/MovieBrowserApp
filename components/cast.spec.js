import React from 'react';
import renderer from 'react-test-renderer';
import Cast from './cast';


const mockedUsedNavigate = jest.fn();

jest.mock('@react-navigation/native', () => ({
  ...jest.requireActual('@react-navigation/native'),
  useNavigation: () => mockedUsedNavigate,
}));

describe('cast test cases', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Cast cast={null} navigation={mockedUsedNavigate} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
