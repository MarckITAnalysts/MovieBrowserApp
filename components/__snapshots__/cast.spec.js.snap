// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`cast test cases renders correctly 1`] = `
<View
  style={
    [
      {
        "marginBottom": 24,
        "marginTop": 24,
      },
    ]
  }
>
  <Text
    style={
      [
        {
          "color": "#fff",
        },
        {
          "fontSize": 18,
          "lineHeight": 28,
        },
        {
          "marginLeft": 16,
          "marginRight": 16,
        },
        {
          "marginBottom": 20,
        },
      ]
    }
  >
    Top Cast
  </Text>
  <RCTScrollView
    contentContainerStyle={
      {
        "paddingHorizontal": 15,
      }
    }
    horizontal={true}
    showsHorizontalScrollIndicator={false}
  >
    <View />
  </RCTScrollView>
</View>
`;
